module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php

```diff
@@ -369,6 +427,13 @@ public function testFormatting(string $function): void
             'c' => 'c',
             'a' => 'a',
             'b' => 'b',
+            'CombineAlt' => 'Standard Title Alternative Title',
+            'CombineAltPrioritizeAlt' => 'Alternative Title Standard Title',
+            'CombineAltDataMethod' => 'Standard Title Other Alternative Title',
+            'CombineAltNoAltFunction' => 'Book',
+            'CombineAltNoAltValue' => '0',
```

Comment:
Not sure I understand why '0' is an appropriate value here -- is it intentional that you mock `getBuildings` and don't provide an expectation with a return value for it? Maybe I'm missing or misunderstanding something...

Reply:
The mock `getBuildings` already existed. I just added `getBuildingsAltScript` to test the behavior when the altScript method exists but does not return anything.

Reply:
Ahh, I see, should have read the comments a little higher up the page. Sorry! :-)